vault/packaging/postinst

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+mkdir -p /var/rudder/plugin-resources
+cp -a /opt/rudder/share/plugins/vault/* /var/rudder/configuration-repository/ncf/
+cd /var/rudder/configuration-repository/ncf/
+rm sample_vault.json
+git add . && git commit -m "Vault plugin installation"
```

Comment:
You should probably mention somewhere in the output that there is a sample_vault.json